Applications/WebPortal/WebApp/library/source/pages/account_directory_page/create_account_page.tsx

```diff
@@ -14,23 +15,61 @@ interface Properties {
 
   /** The database of available countries. */
   countryDatabase: Nexus.CountryDatabase;
+
+  /** The call to submit the profile page. */
+  onSubmit: (username: string, groups: Beam.DirectoryEntry[],
```

Comment:
Callbacks should be optional and go at the bottom. Also document all parameters.